cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -164,7 +164,7 @@ public void execute() {
                 .initConfigurations(this.argList == null ? new String[0] : this.argList.toArray(new String[0]));
 
         // check if there are too many arguments.
-        if (args.length > 1) {
+        if (args.length > 0) {
```

Comment:
Shouldn't the check for > **0** be done only if `-a` is specified?

1 arg is still valid when specifying the file/module name right?

```cmd
$ ballerina build foo
```

Reply:
True, changed the condition

Reply:
We seem to have tests already for this scenario, can we validate if they are running?

Also, we normally use imperative mood for commit messages - please see https://chris.beams.io/posts/git-commit/#imperative

Reply:
@MaryamZi Travis will verify that right ? 
@parkavi11 Lets adhere to the commit message change in future PRs since the build has been running for sometime now.